server_stats.cc

```diff
@@ -7,7 +7,7 @@ void ServerStats::recordInteraction(Request request, Response response) {
   // Increment its count or initialize its count as 0 if never seen before
   const auto it = handler_call_distribution_.find(uri_and_status);
   if (it == handler_call_distribution_.end()) {
-    handler_call_distribution_[uri_and_status] = 0;
+    handler_call_distribution_[uri_and_status] = 1;
```

Comment:
Logic error: when we're recording our first instance of a call, that is 1 call already, not 0!